mne/cuda.py

```diff
@@ -106,19 +106,18 @@ def init_cuda(ignore_config=False):
 ###############################################################################
 # Repeated FFT multiplication
 
-def setup_cuda_fft_multiply_repeated(n_jobs, h_fft):
+def setup_cuda_fft_multiply_repeated(n_jobs, h, n_fft):
```

Comment:
shouldn't this function be private? (ie, the API is changed below ...)

Reply:
Yes maybe I should change them to private. They weren't ever really meant to be public and aren't in our API docs so hopefully it's not too bad to do it